lib/rubocop/cop/rails/rake_environment.rb

```diff
@@ -41,6 +41,16 @@ def on_block(node)
           end
         end
 
+        def autocorrect(node)
+          lambda do |corrector|
+            task_name = node.arguments[0]
+            corrected_task_name = task_name.source.delete(':|\'|"') + ':'
```

Comment:
Ah, this is likely to break with the following task name.

```ruby
% cat Rakefile
task :baz do
  puts 'hello'
end

task 'foo:bar' => :baz

% rake foo:bar
hello
```
https://github.com/rubocop-hq/rubocop-rails/pull/130#issuecomment-533375969

Reply:
Fixed to handle namespaced task names.